ProcessLib/RichardsFlow/Tests.cmake

```diff
@@ -74,19 +74,32 @@ AddTest(
     GLOB RichardsFlow_2d_richardsflow_pcs_0_ts_*.vtu pressure pressure 5e-11 1e-15
     GLOB RichardsFlow_2d_richardsflow_pcs_0_ts_*.vtu saturation saturation 1e-14 1e-15
 )
-#PETSc/MPI
+
 AddTest(
-    NAME 2D_RichardsFlow_h_us_quad_small_PID_adaptive_dt
+    NAME RichardsFlow_square_1e2_flow_fully_saturated
     PATH Parabolic/Richards
-    EXECUTABLE_ARGS RichardsFlow_2d_small_PID_adaptive_dt.prj
-    WRAPPER mpirun
-    WRAPPER_ARGS -np 1
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS flow_fully_saturated.prj
+    WRAPPER time
     TESTER vtkdiff
-    REQUIREMENTS OGS_USE_MPI
-    RUNTIME 220
+    REQUIREMENTS NOT OGS_USE_MPI
     DIFF_DATA
-    ref_t_1600.000000.vtu richards_pcs_0_ts_803_t_1600_000000_0.vtu pressure pressure 1e-8 1e-3
+    GLOB flow_fully_saturated_pcs_0_ts_*.vtu pressure pressure 1e-9 0
```

Comment:
Is this benchmark the same as one of LiquidFLow benchmarks? If it is, please highlight it somewhere, e.g. in the file name or reference variable name.

Reply:
It is same as in RichardsMechanics, which was created based on a RichardsFlow ctest...